src/numbers.js

```diff
@@ -1,45 +1,45 @@
-const add = (a, b) => {
-  // your code here
+const add = (num1, num2) => {
+  return num1 + num2;
 };
 
-const subtract = (a, b) => {
-  // your code here
+const subtract = (num1, num2) => {
+  return num1 - num2;
 };
 
-const multiply = (a, b) => {
-  // your code here
+const multiply = (num1, num2) => {
+  return num1 * num2; 
 };
 
-const divide = (a, b) => {
-  // your code here
+const divide = (num1, num2) => {
+  return num1 / num2;
 };
 
-const power = (a, b) => {
-  // your code here
+const power = (num1, num2) => {
+  return Math.pow(num1, num2);
 };
 
-const round = (a) => {
-  // your code here
+const round = (num1) => {
+  return Math.round(num1);
 };
 
-const roundUp = (a) => {
-  // your code here
+const roundUp = (num1) => {
+  return Math.roundUp(num1);
```

Comment:
`.roundup` is not a method on Math but `.ceil` is, have a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil